changes/1502.incompatible.rst

```diff
@@ -0,0 +1,6 @@
+The 'NotificationReceiver.notifications()' method now continues running when
+there are no notifications, and only ever returns when exceptions are raised.
+Before this change, the method returned when there were no notfications, so it
+had to be invoked by the user in a loop. While such user code would not break
+with this change, it should still be adjusted to accomodate that the method is
+now not expected to ever return.
```

Comment:
There is a second incompatibility: Previously, we did not document that `stomp.exception.StompException` can be raised from several methods.

Reply:
Added